src/audiovideocontroller/DefaultAudioVideoController.ts

```diff
@@ -216,16 +216,18 @@ export default class DefaultAudioVideoController implements AudioVideoController
     this.meetingSessionContext.videosToReceive = new DefaultVideoStreamIdSet();
     this.meetingSessionContext.videosPaused = new DefaultVideoStreamIdSet();
     this.meetingSessionContext.statsCollector = new DefaultStatsCollector(this, this.logger);
+    this.meetingSessionContext.pingPong = new DefaultPingPong(
+      this.meetingSessionContext.signalingClient,
+      DefaultAudioVideoController.PING_PONG_INTERVAL_MS,
+      this.logger
+    );
+    this.meetingSessionContext.pingPong.start();
```

Comment:
Connection monitor is currently [responsible for starting and stopping](https://github.com/aws/amazon-chime-sdk-js/blob/master/src/connectionmonitor/SignalingAndMetricsConnectionMonitor.ts#L43) PingPong. Doing this splits responsibility over PingPong lifecycle.

Reply:
Here is what I understand:
https://github.com/aws/amazon-chime-sdk-js/pull/153#issuecomment-596404883
I might be wrong, please let me know...